ee/app/livechat-enterprise/server/lib/QueueInactivityMonitor.ts

```diff
@@ -0,0 +1,100 @@
+import Agenda from 'agenda';
+import { MongoInternals } from 'meteor/mongo';
+import { Meteor } from 'meteor/meteor';
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
+import moment from 'moment';
+
+import { settings } from '../../../../../app/settings/server';
+import { Logger } from '../../../../../app/logger/server';
+import { LivechatRooms, Users, LivechatInquiry } from '../../../../../app/models/server';
+import { Livechat } from '../../../../../app/livechat/server/lib/Livechat';
+import { IUser } from '../../../../../definition/IUser';
+import { IOmnichannelRoom } from '../../../../../definition/IRoom';
+
+const SCHEDULER_NAME = 'omnichannel_queue_inactivity_monitor';
+
+export class OmnichannelQueueInactivityMonitorClass {
+	scheduler: Agenda;
+
+	running: boolean;
+
+	logger: any;
+
+	_name: string;
+
+	user: IUser;
+
+	message: string;
+
+	constructor() {
+		this.running = false;
+		this._name = 'Omnichannel-Queue-Inactivity-Monitor';
+		this.logger = new Logger('QueueInactivityMonitor');
+		this.scheduler = new Agenda({
+			mongo: (MongoInternals.defaultRemoteCollectionDriver().mongo as any).client.db(),
+			db: { collection: SCHEDULER_NAME },
+			defaultConcurrency: 1,
+		});
+		this.user = Users.findOneById('rocket.cat');
+		this.message = TAPi18n.__('Closed_automatically_chat_queued_too_long');
```

Comment:
Is there a way to select a global language to translate the message? Perhaps a server language or something. This way, English will always be the language used here.

Reply:
Yup, we can get the value for the language setting.